apps/web/ui/partners/risk-review-sheet.tsx

```
@@ -0,0 +1,609 @@
+            </span>
+          </div>
+
+          <div className="space-y-4 pt-4">
+            {parameters && (
+              <div>
+                <h3 className="text-sm font-medium text-neutral-900">
+                  Parameters used
+                </h3>
+                <div className="flex flex-wrap gap-2">
+                  {Object.entries(parameters).map(([key, value]) => (
+                    <div className="inline-flex items-center bg-orange-50">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)